modules/baremetal-event-relay.adoc

```diff
@@ -34,7 +34,7 @@ leaderElection:
 
 [discrete]
 == Project
-link:https://github.com/redhat-cne/hw-event-proxy-operator[baremetal-event-relay]
+link:https://github.com/redhat-cne/hw-event-proxy-operator[hw-event-proxy-operator]
```

Comment:
http://file.emea.redhat.com/aireilly/amolnar-rh_consolidated-ZTP/operators/operator-reference.html#baremetal-event-relay_platform-operators-ref

Reply:
Thanks! Verified.

Reply:
@yliu127 This is not in current PR: http://file.emea.redhat.com/aireilly/amolnar-rh_consolidated-ZTP/monitoring/using-rfhe.html#hw-installing-amq-interconnect-messaging-bus_using-rfhe

because I didn't make any changes to this file for the renaming work...

Reply:
It's in the PR now, I just fixed a typo... modules/hw-installing-amq-interconnect-messaging-bus.adoc